src/bun.zig

```diff
@@ -2754,6 +2754,30 @@ pub inline fn new(comptime T: type, t: T) *T {
     return ptr;
 }
 
+pub inline fn newCatchable(comptime T: type, t: T) error{OutOfMemory}!*T {
```

Comment:
i don't think we should do this for statically known sizes unless you want to write tests for it